apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java

```diff
@@ -201,14 +201,18 @@ public void verifyCorrectEndpointResponse(String endpointId, Supplier<String> en
   }
 
   /**
-   * Generates the "golden" files, i.e. the approved responses to compare to. Run by commenting out
-   * the <code>@Disabled</code> annotation and running this method as JUnit.
+   * Generates the "golden" files, i.e. the approved responses to compare to. Purpose of this
+   * testing is to perform regression testing against the "Golden Beneficiary Data" at a specific
+   * point in time. It is important to note that this testing focuses on checking for regressions
+   * against the data at that particular moment, and not necessarily against data artifacts.
    *
    * @param endpointId the endpoint id
    * @param endpointOperation the endpoint operation
    */
-  @Disabled(
-      "This should not be a test, Should be handled in another location as part of build flow")
+  @EnabledIfSystemProperty(
+      // disabled on all but 64bit OS
+      named = "os.arch",
+      matches = ".*64.*")
```

Comment:
This was an example, but we want an actual environment variable here so we can trigger these tests when its set.

Reply:
As discussed, I've set the system property for the IT tests using Maven and also added a cmd to run the test in the comments section.